bugsnag-plugin-android-anr/src/main/jni/bugsnag_anr.h

```diff
@@ -0,0 +1,16 @@
+/**
+ * JNI interface between bugsnag-plugin-android-anr JVM and C++
+ */
+#ifndef BUGSNAG_ANR_H
+#define BUGSNAG_ANR_H
```

Comment:
Nothing is defined in this file. It should be deleted altogether, moving the one import to `bugsnag_anr.c`